scripts/index.ts

```diff
@@ -0,0 +1,123 @@
+import * as dotenv from "dotenv";
+import { VaultsDocument, execute, Vault } from "../.graphclient";
+import { ethers, providers, BigNumber } from "ethers";
+import { TickMath } from "@uniswap/v3-sdk";
+const { AlchemyProvider } = providers;
+import { readFileSync } from "fs";
+import { getAmountsForLiquidty } from "./liquidity-math";
+import { sqrt } from "./math-utils";
+import { liquidate } from "./bot";
+
+dotenv.config({ path: __dirname + "/.env" });
+
+const main = async () => {
+    const provider = new AlchemyProvider("goerli", process.env.ALCHEMY_KEY);
+    let cdp = new ethers.Contract(
+        process.env.CDP as string,
+        readFileSync("./scripts/abis/Vault.json", "utf-8"),
```

Comment:
minor: maybe human readable ABI from ethers is a bit better and cleaner practice https://docs.ethers.org/v5/api/utils/abi/formats/#abi-formats--human-readable-abi

Reply:
Is there any tool to generate human readable ABI?

Reply:
Don't know, I would write all the methods I need by hand, seems like we need only a few of them. Files are also fine, you can keep them if you want.